### PR TITLE
Translate Codebase Overview

### PR DESCRIPTION
**Pull request by paramithatm**
Translated [Codebase Overview](https://reactjs.org/docs/codebase-overview.html) page.
Please kindly review my work.
Thanks!

### CODE REVIEW

File: content/docs/codebase-overview.md [hunk 1]
```diff
@@ -65,48 +64,48 @@ var didWarnAboutMath = false;
...
+Merupakan hal yang penting untuk menjaga perilaku versi pengembangan dan produksi tetap serupa, sehingga `invariant` dilontarkan baik di mode pengembangan dan produksi. Pesan eror secara otomatis diganti dengan kode eror pada mode produksi untuk menghindari mempengaruhi ukuran byte secara negatif.
 
-### Development and Production {#development-and-production}
+### Pengembangan dan Produksi {#development-and-production}
 
-You can use `__DEV__` pseudo-global variable in the codebase to guard development-only blocks of code.
+Anda dapat menggunakan variabel pseudo-global `__DEV__` pada basis kode untuk menjaga blok kode yang ditujukan hanya pada mode pengembangan.
 
-It is inlined during the compile step, and turns into `process.env.NODE_ENV !== 'production'` checks in the CommonJS builds.
+Ia di-_inline_ pada tahap kompilasi, dan berubah menjadi pengecekan `process.env.NODE_ENV !== 'production'` pada _build_ CommonJS.
```
Comment (resir014) [inline content/docs/codebase-overview.md]:
```suggestion
Variabel ini di-_inline_ pada tahap kompilasi, dan berubah menjadi pengecekan `process.env.NODE_ENV !== 'production'` pada _build_ CommonJS.
```

Less-ambiguous statement without "ia".

File: content/docs/codebase-overview.md [hunk 2]
```diff
@@ -65,48 +64,48 @@ var didWarnAboutMath = false;
...
 
-For standalone builds, it becomes `true` in the unminified build, and gets completely stripped out with the `if` blocks it guards in the minified build.
+Untuk _build_ yang berdiri sendiri, ia menjadi `true` pada _build_ yang tidak di -_minify_, dan dilepas seluruhnya dengan menggunakan blok `if` yang ia jaga pada _build_ yang di-_minify_.
 
 ```js
 if (__DEV__) {
-  // This code will only run in development.
+  // Kode ini hanya akan berjalan di mode.
```
Comment (resir014) [inline content/docs/codebase-overview.md]:
```suggestion
  // Kode ini hanya akan berjalan di mode pengembangan.
```

Did you mean to say this? :)

File: content/docs/codebase-overview.md [hunk 3]
```diff
@@ -117,28 +116,28 @@ ReactRef.detachRefs = function(
...
-When possible, new code should use Flow annotations.
-You can run `yarn flow` locally to check your code with Flow.
+Bila mungkin, kode baru harus menggunakan anotasi Flow.
+Anda dapat menjalankan `yarn flow` secara lokal untuk mengecek kode Anda menggunakan Flow.
 
-### Dynamic Injection {#dynamic-injection}
+### Injeksi Dinamis {#dynamic-injection}
 
-React uses dynamic injection in some modules. While it is always explicit, it is still unfortunate because it hinders understanding of the code. The main reason it exists is because React originally only supported DOM as a target. React Native started as a React fork. We had to add dynamic injection to let React Native override some behaviors.
+React menggunakan injeksi dinamis pada sebagian modul. Walaupun selalu eksplisit, hal tersebut masih disayangkan karena menghalangi pemahaman terhadap kode. Alasan utama ia adalah karena React mulanya hanya mendukung DOM sebagai target. React Native dimulai sebagai _fork_ dari React. Kami perlu menambahkan injeksi dinamis agar React Native dapat melakukan _override_ pada beberapa perilaku.
```
Comment (resir014) [inline content/docs/codebase-overview.md]:
```suggestion
React menggunakan injeksi dinamis pada sebagian modul. Walaupun selalu eksplisit, hal tersebut masih disayangkan karena menghalangi pemahaman terhadap kode. Alasan utamanya adalah karena React mulanya hanya mendukung DOM sebagai target. React Native dimulai sebagai _fork_ dari React. Kami perlu menambahkan injeksi dinamis agar React Native dapat melakukan _override_ pada beberapa perilaku.
```

File: content/docs/codebase-overview.md (hunk 2 above)
Comment (paramithatm) [reply content/docs/codebase-overview.md]: yes, thanks for the correction 😅

**Pull request merged**